components/AppLayout/AppLayout.js

```diff
@@ -42,11 +84,25 @@ function AppLayout({ children }) {
 
   return (
     <Fragment>
-      <AppHeader />
-      {isRouteChanging && <LinearProgress classes={classes} />}
-      <Container>{children}</Container>
+      <AppHeader
+        onMenuButtonClick={toggleSidebar}
+        user={data?.GetUser}
+        openLoginModal={openLoginModal}
+        logout={logout}
```

Comment:
I think using `onLogout={logout}` / `onLoginModalOpen={openLoginModal}` adheres to the convention more. The same goes to `AppSidebar`.